src/onefetch/cli.rs

```diff
@@ -73,6 +74,13 @@ impl Cli {
                         .collect::<Vec<&str>>()
                 ),
         )
+        .arg(
+            Arg::with_name("ascii-input")
+                .long("ascii")
+                .value_name("STRING")
+                .takes_value(true)
```

Comment:
We probably need to add a `.max_values(1)` and also provide a `.validator` to make sure the ascii-input isn't empty. Indeed, `onefetch --ascii ""` results in:

```
thread 'main' panicked at 'assertion failed: self.end >= self.start', src/onefetch/ascii_art.rs:41:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```
And als maybe a `.long_help` with an example: `onefetch --ascii "$(fortune | cowsay -W 25)"`

Reply:
Thanks for taking a look, I totally forgot to try out an empty string ^^

The new `.validator` wont allow empty strings. I also added a `.long_help`, but I didn't know if it should have more detail or not since its the only `.long_help` at the moment and I had nothing to compare to.